src/libopensc/card-myeid.c

```diff
@@ -131,9 +135,13 @@ static int myeid_init(struct sc_card *card)
 	u8 defatr[SC_MAX_ATR_SIZE];
 	size_t len = sizeof(defatr);
 	const char *atrp = myeid_atrs[MYEID_INFINEON_CHIP_ATR];
-
+	
 	LOG_FUNC_CALLED(card->ctx);
+
+	card->name = myeid_card_name;
```

Comment:
Take a look at myeid_get_info, it is used there again. This change fixed an actual problem that resulted in garbage being displayed with command opensc-tool -n